File: Objective-C/WCDB_Test/Person+CoreDataProperties.m
//
//  Person+CoreDataProperties.m
//  WCDB_Test
//
//  Created by 姜泽东 on 2017/9/15.
//  Copyright © 2017年 MaiTian. All rights reserved.
//

#import "Person+CoreDataProperties.h"

@implementation Person (CoreDataProperties)

+ (NSFetchRequest<Person *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"Person"];
}

@dynamic name;
@dynamic sex;
@dynamic work;
@dynamic age;
@dynamic height;
@dynamic telephone;

@end

File: Swift/iOS10SwiftCoreData/iOS10SwiftCoreData/Person+CoreDataProperties.swift
//
//  Person+CoreDataProperties.swift
//  iOS10SwiftCoreData
//
//  Created by zidonj on 2016/11/9.
//  Copyright © 2016年 zidon. All rights reserved.
//

import Foundation
import CoreData


extension Person {

    @nonobjc public class func fetchRequest() -> NSFetchRequest<Person> {
        return NSFetchRequest<Person>(entityName: "Person");
    }

    @NSManaged public var name: String?
    @NSManaged public var age: String?
    @NSManaged public var height: String?

}
